reference/providers/kubernetes-v2/index.md

```diff
@@ -200,5 +200,20 @@ be rendered as well.
 
 ## Services, Ingresses
 
+| __Resource__ | _Deploy_ | _Delete_ | Versioned | Stability |
+|-|:-:|:-:|:-:|-|
+| __`Service`__ | Yes | Yes | No | The `status.loadBalancer` field reports that a load balancer was found if and only if the service type is `LoadBalancer`. |
+| __`Ingress`__ | Yes | Yes | No | The `status.loadBalancer` field reports that a load balancer was bound. |
+
 ## NetworkPolicies
 
+| __Resource__ | _Deploy_ | _Delete_ | Versioned | Stability |
+|-|:-:|:-:|:-:|-|
+| __`NetworkPolicy`__ | Yes | Yes | No | Automatically "stable". |
+
+## ConfigMaps, Secrets
+
+| __Resource__ | _Deploy_ | _Delete_ | Versioned | Stability |
+|-|:-:|:-:|:-:|-|
+| __`ConfigMap`__ | Yes | Yes | Yes | Automatically "stable". |
+| __`Secret`__ | Yes | Yes | Yes | Automatically "stable". |
```

Comment:
I wonder if it might be better, instead of putting "stable" in double-quotes, linking it to the __Stability__ heading under __Resource Management Policies__, or to __Accomodating Level-Based Deployments__, where you have your statement, "A deployed manifest is considered stable when..." In fact, would it make sense to move that definition of stability to under the aforementioned Stability heading.

Reply:
BTW, "Accomodating" in that heading is misspelled.

Reply:
SGTM